client/src/app/s-degree-plan-editor/s-degree-plan-editor.component.html

```diff
@@ -230,28 +244,34 @@
         </div>
         <div class="add-year-card">
           <button
-            *ngIf="plan.isAddingYear === false"
+            *ngIf="addYearWidget.active === false"
             class="add-semester-font-clr w100 h100"
             mat-button
             (click)="toggleAddingYear()"
           >
             + ADD A NEW SCHOOL YEAR
           </button>
-          <mat-form-field *ngIf="plan.isAddingYear === true">
+          <mat-form-field *ngIf="addYearWidget.active === true">
```

Comment:
change to just ngIf="addYearWidget.active" because you are saying if (true == true) when you could say if(true)

Reply:
Changed